risc0-adapter/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "risc0-adapter"
```

Comment:
Doe is make sense to create `adapters/risc0` , this way there will be a natural place to add other adapters?

Reply:
Good idea. Moved!